libs/sdk-utils/multicall/src/examples/index.ts

```diff
@@ -0,0 +1,59 @@
+import { BigNumber, ethers } from 'ethers';
+import { Erc20Abi as ERC20 } from '@shared/util-blockchain';
+import { erc20 } from '@shared/util-blockchain/abis';
+import { MultiCallWrapper } from '../lib/sdk-utils-multicall';
+import { promiseObject } from '@shared/helpers';
+
+// 🐋🐋🐋🐋🐋🐋🐋🐋🐋
+const USDCholders = [
+  '0x95bf7e307bc1ab0ba38ae10fc27084bc36fcd605',
+  '0x12edea9cd262006cc3c4e77c90d2cd2dd4b1eb97',
+  '0xe61dd9ca7364225afbfb79e15ad33864424e6ae4',
+  '0x2b4c76d0dc16be1c31d4c1dc53bf9b45987fc75c',
+  '0x083dee8e5ca1e100a9c9ec0744f461b3507e9376',
+  '0xcb54ea94191b280c296e6ff0e37c7e76ad42dc6a',
+];
+
+// Use a provider other than default (i.e. with FTM)
+export async function main() {
```

Comment:
Is this leftover code?

Reply:
No I actually added it as some worked examples with specific cases you might need. I personally love examples so here is one that might reflect a typical "I need to fetch multiple balances with a custom provider"

Reply:
I see. I'd move this into the README file, but I don't mind it being here either. It is just something what won't be read by 3rd parties who usually either read the API docs or the README.